app/javascript/components/lease_ups/shortFormActions.js

```diff
@@ -0,0 +1,4 @@
+import apiService from '~/apiService'
+
+export const getShortFormApplication = async (applicationId) =>
```

Comment:
I think after the dust settles on the routing/state work we should think about how to split up our actions and apiService request functions in a more reasonable way